src/js/constants/mocks/useToast-data.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+
+import Toaster from "@constants/useToast-data";
+
+namespace Toastify {
+  export const TestMarkupMessage = 'My <b>test</b> message.';
+  export const TestTypeOption: Toaster.Option = { type: 'success' };
+  export const TestClassListOption: Toaster.Option = { type: 'warning', classlist: 'text-warning bg-dark border-1' };
+  export const TestAutoHideOption: Toaster.Option = { type: 'danger', autohide: false };
+
+  export const ExpectedTypeClassList = Toaster.Theme[Toastify.TestTypeOption.type];
+
+  export const DefaultToastType = Toaster.Default.type;
+  export const FallbackContainerClass = 'toast-container--fallback';
+
+  export const WithContainerWithTemplate = `
+    <div class="toast-container position-fixed top-0 end-0 p-3" id="js-toast-container">
+      <template class="js-toast-template">
+        <div class="toast" role="alert" aria-live="assertive" aria-atomic="true">
+          <div class="d-flex">
+            <div class="toast-body"></div>
+            <button type="button" class="btn-close me-2 m-auto d-none" data-bs-dismiss="toast"></button>
+          </div>
+        </div>
+      </template>
+    </div>
+    `;
```

Comment:
Add this variable into something like `prestashop.templates.useToast.toast`

so it's overridable from another JS file

Same for the container bellow: `prestashop.templates.useToast.container`

Verify if `prestashop.templates` exist before, overwise you'll need to add it to `prestashop` before

Reply:
Well, sorry, it's the wrong file, I was speaking about the not mocked data, the real fallback

Reply:
Why a JS file should be able to override test data?

Reply:
https://github.com/PrestaShop/theme-refacto/pull/184#discussion_r813754515

:trollface:

Reply:
All data have been defined as const.
I think these constants must be managed by useToast only.

Developers can use their templates to style the toast, just need to have standard structure of BS Toast in tpl file.
For example they can add header for toast.
The JS files can use options for overriding styles.

Reply:
If it's needed a JS file uses its template for toast, we can add a parameter to useToast for custom template:
`const useToast = (message: string, options?: Toaster.Option, template?: string)`